mlrun/config.py

```diff
@@ -94,6 +94,12 @@
     "datastore": {"async_source_mode": "disabled"},
     # default node selector to be applied to all functions - json string base64 encoded format
     "default_function_node_selector": "e30=",
+    # default priority class to be applied to functions running on k8s cluster
+    "default_function_priority_class_name": "",
+    # valid options for priority classes - separated by a comma
+    # empty string means nothing is valid, will use default_function_priority_class_name if present
+    # else will not support pods priority
```

Comment:
Are you sure default with empty valid list will work ? I think it will indeed get the default set on the client side, but then won't pass validation on server side (which is ok, and means that just your comment is wrong and should be `empty string means nothing is valid, i.e. no option to use pods priority`

Reply:
the default value is not going through validation unless the user executes `with_priority_calls_name` with the default value because that's the only place we do this validation. worth discussing if default value should only work if it is in the valid list.